openslides_backend/action/actions/user/update.py

```diff
@@ -47,7 +47,6 @@ class UserUpdate(
             "gender",
             "email",
             "default_number",
-            "default_structure_level",
```

Comment:
Why didn't you add structure_level_ids? You did in create_update_permissions.

Reply:
`default_structure_level` was removed completely without replacement. `structure_level` was replaced with `structure_level_ids`, which are present a few lines below in the `UserMixin.transfer_list`.